docs/model-archiver.md

```diff
@@ -159,4 +159,4 @@ For eagermode:
 torch-model-archiver --model-name densenet_161 --version 1.0 --model-file model.py --serialized-file model.pt --handler image_classifier
 ```
 
-This will package all the model artifacts files and output `densenet_161.mar` in the current working directory. This `.mar` file is all you need to run TorchServe, serving inference requests for a simple image recognition API. Go back to the [Serve a Model tutorial](../README.md#serve-a-model) and try to run this model archive that you just created!
+This will package all the model artifacts files and output `densenet_161.mar` in the current working directory. This `.mar` file is all you need to run TorchServe, serving inference requests for a simple image recognition API. Go back to the [Serve a Model tutorial](https://github.com/pytorch/serve/blob/master/README.md#serve-a-model) and try to run this model archive that you just created!
```

Comment:
This should link to the same thing you have on index.rst: 
```
:ref:`Serving Quick Start <install:Serve a model>` - Basic server usage tutorial
```

Reply:
I think that syntax only works for .rst files. I'm not aware of a way to link to s subheader in another file in a .md file. 
The alternatives I would suggest are:
* `[Serve a Model tutorial](install.md)`
* `[Serve a Model tutorial](https://pytorch.org/serve/docs/sphinx/build/html/install.html)`